SampleGoGrpc/SimpleMath/grpc.go

```diff
@@ -0,0 +1,259 @@
+//Generated by GRPC Go plugin
+package SimpleMath
+
+import "github.com/google/flatbuffers/go"
+
+import (
+  grpc "google.golang.org/grpc"
+  context "golang.org/x/net/context"
+)
+
+// FlatCodec implements Grpc-go Codec
+
+type FlatCodec struct{}
+
+func (FlatCodec) Marshal(v interface{}) ([]byte, error) {
+  return v.(*flatbuffers.Builder).FinishedBytes(), nil
+}
+
+func (FlatCodec) Unmarshal(data []byte, v interface{}) error {
+  v.(flatbuffersInit).Init(data, flatbuffers.GetUOffsetT(data))
+  return nil
+}
+
+func (FlatCodec) String() string {
+  return "flatbuffers"}
```

Comment:
Because this is a constant, should we lift it up into a global variable instead? That way users don't have to use string equality on it.